### PR TITLE
Add github files

### PR DESCRIPTION
**Pull request by camilamacedo86**
## Motivation
Add github files in order to follow our standards and the good practice to work with opensource.  

## What
Add github files 

## Why
- Following the [opensource guide](https://opensource.guide/) recommendations and checklists for contributors choose a project to contribute with
- Following our standards
- Help the users/contributor track properly an issue and a PR

## How
Applying the same ones defined in: https://github.com/integr8ly/installation/tree/master/.github

### CODE REVIEW

File: .github/iSSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+<!-- Describe the current/actual behaviour. -->
+
+## Environment
+
+* Operating system: (E.g RHEL 7.6 )
+* OpenShift version:
+<!-- Run the command `oc version` and add the result here. -->
+* Ansible version:
+<!-- Run the command `ansible --version` and add the result here. -->
+* Project Version/Tag: (E.g release-1.0.1)
```
Comment (david-martin) [inline .github/iSSUE_TEMPLATE.md]: @camilamacedo86 As this is a node.js app, I think asking for the Node.js & NPM version is sufficient here.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+## Motivation
+<!-- Add references to relevant tickets or a short description of what motivated you to do it. (E.g Trello Card AND/OR ISSUE: https://github.com/integr8ly/tutorial-web-app/issues) -->
```
Comment (david-martin) [inline .github/PULL_REQUEST_TEMPLATE.md]: should this example have a link to this repo?

**Comment by camilamacedo86** [comment]
@david-martin the changes requested are done. Please, could you check if it is missing anything else?

File: .github/iSSUE_TEMPLATE.md (hunk 1 above)
Comment (david-martin) [reply .github/iSSUE_TEMPLATE.md]: @camilamacedo86 Apologies, I was thinking of a different repo.
THis is a Java project, so Java & Maven should be fine here

**Comment by camilamacedo86** [comment]
Hi @david-martin,

Tks. The change was made  and I am merging it. Please, feel free to check and update if required.

**Pull request merged**